src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -10,10 +10,12 @@
          to be trimmed when Invariant=true, and allows for the Settings static cctor (on Unix) to be preserved when Invariant=false. -->
     <type fullname="System.Globalization.GlobalizationMode">
       <method signature="System.Boolean get_Invariant()" body="stub" value="true" feature="System.Globalization.Invariant" featurevalue="true" />
+      <method signature="System.Boolean get_Hybrid()" body="stub" value="true" feature="System.Globalization.Hybrid" featurevalue="true" />
       <method signature="System.Boolean get_PredefinedCulturesOnly()" body="stub" value="true" feature="System.Globalization.PredefinedCulturesOnly" featurevalue="true" />
     </type>
     <type fullname="System.Globalization.GlobalizationMode/Settings">
       <method signature="System.Boolean get_Invariant()" body="stub" value="false" feature="System.Globalization.Invariant" featurevalue="false" />
+      <method signature="System.Boolean get_Hybrid()" body="stub" value="false" feature="System.Globalization.Hybrid" featurevalue="false" />
```

Comment:
We should have someone more familiar with the linker stuff look over these parts.
Can we check what gets trimmed for the various feature switches, and ensure the trimming tests pass.
I'm actually not sure if we need this line. I *think* that `get_Invariant` is substituted as false when the feature switch is off specifically to preserve this https://github.com/dotnet/runtime/blob/d260e5023b6f1593fb38534c81ff030805e9ae0f/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs#L15-L34

Since we don't have a need for `GlobalizationMode.Hybrid` in that constructor, we probably don't need this (and could be the reason that `get_PredefinedCulturesOnly` isn't added to this section either.

Reply:
Makes sense, I will remove for now.